rtvi-client-android-daily/src/androidTest/java/ai/rtvi/client/daily/MainTests.kt

```diff
@@ -28,25 +29,28 @@ import org.junit.Test
 class MainTests {
 
     companion object {
-        private val options = VoiceClientOptions(
+        private val options = RTVIClientOptions(
```

Comment:
Should we create an alias to mark `VoiceClientOptions` as deprecated ?

Reply:
But I guess that means we would need to change a couple of places if we want to maintain backwards compatibility.